include/seqan3/alignment/pairwise/align_result.hpp

```diff
@@ -34,210 +34,154 @@
 
 /*!\file
  * \brief Provides seqan3::align_result.
- * \author Rene Rahn <rene.rahn AT fu-berlin.de>
```

Comment:
Oh, I didn't remove you on purpose. I created a new file from scratch and forgot to include you again – sorry!! 
I see it looks weird, because it shows the diff of the old and new file (with only a few brackets matching).